tests/integration-tests/tests/schedulers/test_custom_munge_key.py

```diff
@@ -48,24 +49,35 @@ def test_custom_munge_key(
     7. Roll back with failure: Update cluster to add back the custom munge key. But let the cluster update fail after
        the custom munge key has been added. Trigger cluster roll back. Test if munge key is fully functional.
     """
-    encode_custom_munge_key = create_base64_encoded_key()
+    encoded_custom_munge_key = create_base64_encoded_key()
     custom_munge_key_arn = store_secret_in_secret_manager(
         region,
-        secret_string=encode_custom_munge_key,
+        secret_string=encoded_custom_munge_key,
     )
     cluster_config = pcluster_config_reader(custom_munge_key_arn=custom_munge_key_arn)
     cluster = clusters_factory(cluster_config, upper_case_cluster_name=True)
 
+    # 1. Deployment verification: Confirm munge key is successfully shared across head, compute and login nodes,
+    # and jobs can be submitted.
+
     # Test if the munge key was successfully fetched, decoded and shared in HeadNode and LoginNodes
     remote_command_executor = RemoteCommandExecutor(cluster)
-    _test_custom_munge_key_fetch_and_decode(remote_command_executor, encode_custom_munge_key)
+    check_encoded_munge_key(remote_command_executor, encoded_custom_munge_key)
```

Comment:
Please put an underscore in front of the name of this function.

Reply:
Done!